how-to-make-a-pitch-into-a-blogpost/readme.md

```diff
@@ -0,0 +1,46 @@
+>**Goal of this guide:** Explaining how to divide the process from idea to end result in to small workable steps that are all individually publishable articles.
+
+# How to turn a pitch into a publishable auricle
+In this guide I will try to explain how to divide the process from idea to end result in to small workable steps that are all individually publishable articles.
```

Comment:
auricle -> article
in to -> into

" are all individually publishable articles."
Yes.. if you write a bit it should stand on it's own if there's nothing created yet. However if something has already been created the end result should always be publishable but the addition itself does not necessarily have to be.

I would remove: 'i will try to .. '
There is no I because the guide is incrementally updated by everyone